modules/RTC/CodecSelection.spec.js

```diff
@@ -42,7 +40,6 @@ class MockConference extends Listenable {
         };
 
         this.activeMediaSession = undefined;
-        this.eventEmitter = new EventEmitter();
```

Comment:
Suggestion: lets use `this.emit` instead of `this.eventEmitter.emit` in this class? WDYT?